azext_edge/edge/providers/support/billing.py

```diff
@@ -0,0 +1,106 @@
+# coding=utf-8
+# ----------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License file in the project root for license information.
+# ----------------------------------------------------------------------------------------------
+
+from functools import partial
+from typing import Iterable, List
+
+from azext_edge.edge.common import PodState
+from knack.log import get_logger
+
+from ..edge_api import BILLING_API_V1, EdgeResourceApi
+from .base import (
+    DAY_IN_SECONDS,
+    assemble_crd_work,
+    process_cron_jobs,
+    process_deployments,
+    process_jobs,
+    process_replicasets,
+    process_services,
+    process_v1_pods,
+)
+
+logger = get_logger(__name__)
+
+AIO_BILLING_USAGE_NAME_LABEL = "app.kubernetes.io/name in (microsoft-iotoperations)"
+ARC_BILLING_EXTENSION_COMP_LABEL = "app.kubernetes.io/component in (billing-operator)"
+
+
+def fetch_pods(
+    since_seconds: int = DAY_IN_SECONDS,
+) -> List[PodState]:
```

Comment:
Is this the reason for changing podstate to listable? is it even needed?

Reply:
yea is there an issue with List[dict] for this type hint?

Reply:
good catch, I think I changed something and forgot to change back, will update